### PR TITLE
FixerFactoryTest - test that priority pair fixers have itest

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

**Comment by keradus** [comment]
@localheinz could you please take a look on this AutoReview ?

File: tests/AutoReview/FixerFactoryTest.php [hunk 1]
```diff
@@ -199,4 +214,79 @@ function ($name) {
 
         return $cases;
     }
+
+    /**
+     * @dataProvider provideFixersPriorityParisHaveItestCases
+     */
+    public function testFixersPriorityParisHaveItest(FixerInterface $first, FixerInterface $second)
```
Comment (ntzm) [inline tests/AutoReview/FixerFactoryTest.php]: Typo: paris -> pairs

File: tests/AutoReview/FixerFactoryTest.php [hunk 2]
```diff
@@ -260,12 +260,13 @@ public function testFixersPriorityPairsHaveIntegrationTest(FixerInterface $first
 
     public function provideFixersPriorityPairsHaveIntegrationTestCases()
     {
+        $self = $this;
```
Comment (localheinz) [inline tests/AutoReview/FixerFactoryTest.php]: Not sure if there's a convention on how to do this on PHP 5.3, but I myself have used (and seen) others use

```php
$that = $this;
```

Maybe using `$self` is a bit confusing?

Comment (keradus) [reply tests/AutoReview/FixerFactoryTest.php]: I saw both conventions, no hard feeling about it

Comment (julienfalque) [reply tests/AutoReview/FixerFactoryTest.php]: `var that = this` is common in Javascript. Also, `$self` could be confused with `self`. I would go for `$that`.

Comment (keradus) [reply tests/AutoReview/FixerFactoryTest.php]: > `var that = this` is common in Javascript

`that` is disrespected by Crockford, he asked you to use self ;P

we gonna drop it in very next branch...

Comment (SpacePossum) [reply tests/AutoReview/FixerFactoryTest.php]: :+1: for `$self` , 
I get confused when `this` is referred by `that`

Comment (keradus) [reply tests/AutoReview/FixerFactoryTest.php]: guys, let go anyway here, I'm fine with both, if any of you wants - let you just change it yourself ;)
Please, let us focus on real idea behind this PR instead ;)

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Comment by keradus** [comment]
replaced by #3425

**Pull request merged**